plugins/BEdita/API/src/Auth/EndpointAuthorize.php

```diff
@@ -123,12 +135,13 @@ public function authorize($user, ServerRequest $request)
      * Perform user unauthentication to return 401 Unauthorized
      * instead of 403 Forbidden
      *
-     * @return void
+     * @return mixed
      */
     protected function unauthenticate()
     {
         $controller = $this->_registry->getController();
-        $controller
+
+        return $controller
```

Comment:
`BaseAuthenticate::unauthenticated()` method must not return any value, according to its documentation. This return is not needed, and not used, as far as I can tell. 🤔